pkg/reconciler/v1alpha1/autoscaling/kpa/kpa_test.go

```diff
@@ -559,22 +537,14 @@ func TestNonKPAClass(t *testing.T) {
 	servingClient.AutoscalingV1alpha1().PodAutoscalers(testNamespace).Create(kpa)
 	servingInformer.Autoscaling().V1alpha1().PodAutoscalers().Informer().GetIndexer().Add(kpa)
 
-	reconcileCh := make(chan error)
-	go func() {
-		defer close(reconcileCh)
-		if err := ctl.Reconciler.Reconcile(context.Background(), testNamespace+"/"+testRevision); err != nil {
-			reconcileCh <- err
-		}
-	}()
-
-	// Wait for reconcile to finish
-	select {
-	case err, ok := <-reconcileCh:
-		if ok && err != nil {
-			t.Errorf("Reconcile() = %v", err)
-		}
-	case <-time.After(3 * time.Second):
-		t.Fatal("Reconciliation timed out")
+	reconcileGrp := errgroup.Group{}
```

Comment:
Sorry I missed this earlier - Do we need this errgroup at all since its only doing one goroutine? i.e. can we just run `ctl.Reconciler.Reconcile(context.Background(), testNamespace+"/"+testRevision)`

Reply:
Very good point, I missed that. Fixed!